projects/plugins/social/src/js/components/admin-page/index.jsx

```diff
@@ -23,36 +22,36 @@ import ToggleSection from './../toggle-section';
 import InfoSection from './../info-section';
```

Comment:
I don't think this is strictly to do with this PR, but these import paths can just be `../` rather than `./../`